lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -326,6 +337,38 @@ withNetworkLayer tr network np project k = do
         runBFM bfConfig $ liftEither . fromBlockfrostPP networkId
             =<< BF.getLatestEpochProtocolParams
 
+    currentSlottingParameters :: BF.ClientConfig -> IO SlottingParameters
+    currentSlottingParameters bfConfig = runBFM bfConfig do
+        BF.EpochInfo{_epochInfoEpoch} <- BF.getLatestEpoch
+        epochNo <- fromBlockfrostM _epochInfoEpoch
+        let EraParams{..} = currentEraParams epochNo
+        epochLen <- unEpochSize eraEpochSize <?#> "EpochSize"
+        securityParameter <- case eraSafeZone of
+            StandardSafeZone wo -> Quantity <$> wo <?#> "StandardSafeZone"
+            UnsafeIndefiniteSafeZone -> error "UnsafeIndefiniteSafeZone"
+        pure SlottingParameters
+            { getSlotLength = SlotLength $ ST.getSlotLength eraSlotLength
+            , getEpochLength = EpochLength epochLen
+            , getActiveSlotCoefficient = ActiveSlotCoefficient 0.05
+            -- TODO: what's better than hardcoding it          ^^^^ ?
```

Comment:
> -- TODO: what's better than hardcoding it          ^^^^ ?

A: Reading it from the genesis file would be the natural choice, but I believe that hardcoding is perfectly fine, given that we already harcode other stuff about mainnet and testnet.

Blockfrost even provides this information

```hs
BF._genesisActiveSlotsCoefficient <$> BF.getLedgerGenesis
```